src/parser.rs

```diff
@@ -558,6 +633,42 @@ mod tests {
         }
     ];
 
+    parse_let_statement_test_case![
```

Comment:
We need negative test cases for parsing, that checks errors are as expected. I guess for that we also need `Result`s rather than `panic`s. Ho hum

Reply:
Will address in a follow up patch.

Reply:
> Will address in a follow up patch.

I'm working on the errors as we speak